lib/browser/guest-view-manager.js

```diff
@@ -340,35 +340,35 @@ const isWebViewTagEnabled = function (contents) {
   return isWebViewTagEnabledCache.get(contents)
 }
 
-const handleMessage = function (channel, handler) {
-  ipcMainInternal.on(channel, (event, ...args) => {
-    if (isWebViewTagEnabled(event.sender)) {
-      handler(event, ...args)
-    } else {
-      event.returnValue = null
-    }
-  })
+const assertWebViewTagEnabled = function (event) {
```

Comment:
Passing some generic `event` here looks somewhat confusing.
Can you please pass `event.sender` when you call this function?, and call the function argument, let's say, `webContents`?